atom/epistula/README.md

```diff
@@ -0,0 +1,75 @@
+# Implementation of the epistula protocol
+Official documentation for the epistula protocol can be found [here](https://epistula.sybil.com/).
+
+## Quick Start
+
+```python
+from atom.epistula.epistula import Epistula
+from substrateinterface import Keypair
+
+# Initialize
+epistula = Epistula()  # Default 8000ms timeout
+# or 
+epistula = Epistula(allowed_delta_ms=10000)  # Custom timeout
+
+# Create message
+message = {"data": "Hello"}
+body = epistula.create_message_body(message)
+
+# Generate headers
+sender = Keypair.create_from_uri('//Alice')
+headers = epistula.generate_header(
+    hotkey=sender,
+    body=body
+)
+
+# Verify signature
+result = epistula.verify_signature(
+    signature=headers["Epistula-Request-Signature"],
+    body=body,
+    timestamp=headers["Epistula-Timestamp"],
+    uuid=headers["Epistula-Uuid"],
+    signed_by=headers["Epistula-Signed-By"]
```

Comment:
question: could you just do `verify_signature(body = body, **headers)`?

Reply:
No because the keys of the headers have different names from the fields in the function :/